src/library/dlgtrackinfo.cpp

```diff
@@ -43,17 +43,43 @@ void DlgTrackInfo::init() {
 
     coverBox->insertWidget(1, m_pWCoverArtLabel);
 
-    if (m_bNavigationIsEnabled) {
-        connect(btnNext, &QPushButton::clicked, this, &DlgTrackInfo::slotNext);
-        connect(btnPrev, &QPushButton::clicked, this, &DlgTrackInfo::slotPrev);
+    m_pTagFetcher.reset(new DlgTagFetcher(this, m_pTrackModel));
+
+    if (m_pTrackModel) {
+        connect(btnNext,
+                &QPushButton::clicked,
+                [=]() { slotNext(); });
```

Comment:
There's no need for a lambda here. Just use `&DlgTrackFetcher::slotNext`

Reply:
slotNext has to be called with a lambda as well because QPushButton::clicked sends false by default, which overrides the default parameter in slotNext.

Reply:
Oh okay. I've come across this before where a signal has a parameter that the slot doesn't need. Please add a comment explaining this.